new-frontiers-14/frontier-station-14
# Move tank harness to outer clothing

## About the PR



Moves tank harness to outer clothing rather than being hidden in their own category.

## Why / Balance


#2022.

## How to test


1. Create a new vox character.
2. Swap between jobs in character setup.
3. Your outfit should include your weird tank harness.

## Media


Default pirate captain outfit.  Fearsome.
![image](https://github.com/user-attachments/assets/7eab7a6a-513b-4754-bf8b-9d4e4377156c)

Default valet outfit.  Nice arm holes.
![image](https://github.com/user-attachments/assets/d15a8ad4-5c5b-4773-9b87-6608bf0d5c7c)

## Requirements


## Breaking changes


**Changelog**



:cl:
- fix: The tank harness for Vox is now in the outer clothing option.  Double check your loadouts.
